src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -239,12 +239,12 @@ public static function phpize($value)
                 $raw = $value;
                 $cast = (int) $value;
 
-                return '0' == $value[0] ? octdec($value) : (((string) $raw === (string) $cast) ? $cast : $raw);
+                return '0' == $value[0] ? octdec($value) : (($raw === (string) $cast) ? $cast : $raw);
```

Comment:
`$value = (string) $value;` and then `$raw = $value;` so `$raw` is always a string